### DISCUSSION TITLE
Dynamic MongoDB name for user auth

### DISCUSSION THREAD

**Discussion by AleksandrukTad** [nextauthjs/next-auth#3508] [Help]
Hello,
I am trying to build multi tenant system based on the subdomain so: companyA.<domain>.com / companyB.<domain>.com etc.
I would like to have users in each database. Is there a way to dynamically change in which DB NextAuth is creating user tables?

**Comment by balazsorban44** [comment]
If you have a way of identifying which domain the request is coming from, you can use advanced initialization and intercept the request:

https://next-auth.js.org/configuration/initialization#advanced-initialization
